inst/rmarkdown/templates/thesis/skeleton/skeleton.Rmd

```diff
@@ -45,6 +45,7 @@ bibliography: [bib/packages.bib,bib/references.bib]
 bibliography-heading-in-pdf: References
 bib-humanities: true   #set to true if you want in-text references formatted as author-year
 doi-in-bibliography: true #set to true if you want DOI's to be shown in the bibliography
+# csl: apa.csl #Uncomment to integrate the bibliography style file that you have put in the root folder of your thesis. Search the Zotero Stlye Repository for the required citation style (https://www.zotero.org/styles).
```

Comment:
This is great. Does it work via pandoc or the LaTeX template?

Let's preselect one intext citation style and one footnote citation style as options for different departments for now. Can you create a shortlist?

Reply:
It works via Pandoc. The user can specify the desired citation style by referencing a csl file in the bibstyles folder (just updated it, it's no longer the root directory) in the YAML header of the index file. 

I preselected the following bibliography styles and downloaded their latest version in the bibstyles file. 
- APA (for social sciences)
- MLA (for humanities)
- Chicago author-date (for social sciences)
- Chicago full note (for history)

Let me know if there is another citation style I should include by default.